src/utils/transaction.ts

```diff
@@ -140,13 +141,27 @@ const transaction = {
         continue;
       }
       const xpriv = xprivkey.deriveNonCompliantChild(addressInfo.bip32AddressIndex);
+      signatures.push({
+        inputIndex,
+        addressIndex: addressInfo.bip32AddressIndex,
+        signature: this.getSignature(dataToSignHash, xpriv.privateKey),
+        pubkey: xpriv.publicKey.toDER(),
```

Comment:
We used to have `xpriv.publicKey.toBuffer()` for this. Is there any difference between both codes?

Reply:
No, internally `toBuffer()` is the same method as `toDER()`.
[reference](https://github.com/bitpay/bitcore/blob/v10.0.28/packages/bitcore-lib/lib/publickey.js#L327)